phpBB/includes/functions_content.php

```diff
@@ -1283,7 +1285,18 @@ function parse_attachments($forum_id, &$message, &$attachments, &$update_count_a
 
 				// Macromedia Flash Files
 				case ATTACHMENT_CATEGORY_FLASH:
-					list($width, $height) = @getimagesize($filename);
+					try
+					{
+						$file_info = $storage_attachment->file_info($filename);
+
+						$width = $file_info->image_width;
```

Comment:
https://tracker.phpbb.com/browse/PHPBB3-10919
Probably this isn't a bad idea, or at least it should be stored in cache or something
With local storage is ok, but with remote storages could be a problem

Reply:
I think we should actually think of dropping flash as well.

Reply:
Yes, but maybe in other PR
About this, i created a ticket to reduce the storage api calls